api/geth_backend.go

```diff
@@ -886,29 +886,6 @@ func (b *GethStatusBackend) AppStateChange(state string) {
 	b.log.Info("App State changed", "new-state", s)
 	b.appState = s
 
-	if s == appStateBackground {
```

Comment:
This is not needed anymore, we always starts/stop wallet from the client at the moment